op-node/rollup/derive/engine_update.go

```diff
@@ -150,6 +152,18 @@ func confirmPayload(
 	if err := sanityCheckPayload(payload); err != nil {
 		return nil, BlockInsertPayloadErr, err
 	}
+	if envelope.BlobsBundle != nil && len(envelope.BlobsBundle.Blobs) > 0 {
+		// When updateSafe, it must be deriving instead of sequencing. Deriving is based on onchain-data which doesn't contain L2 blob.
+		if updateSafe {
```

Comment:
I couldn’t find any logic where the op-nodes attempt to download blobs from the DAC to confirm that BLOBs are available during derivation. Should we consider implementing this? It would be helpful to have a specification outlining how we want to implement the entire feature.

Reply:
Currently l2 blocks are stored at da server, users can download the blob by blobhash if they need it.